src/app/[locale]/vulnerabilities/detail/[id]/components/Summary.tsx

```diff
@@ -0,0 +1,184 @@
+// Copyright (C) Siemens AG, 2023. Part of the SW360 Frontend Project.
+
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+
+// SPDX-License-Identifier: EPL-2.0
+// License-Filename: LICENSE
+
+'use client'
+
+import { VulnerabilitySummaryDataType } from '@/object-types/VulnerabilitySummaryDataType'
+import { useTranslations } from 'next-intl'
+import { COMMON_NAMESPACE } from '@/object-types/Constants'
+import { useState } from 'react'
+import Link from 'next/link'
+import styles from '../detail.module.css'
+
+export default function Summary({ summaryData }: { summaryData: VulnerabilitySummaryDataType }) {
+    const t = useTranslations(COMMON_NAMESPACE)
+    const [toggle, setToggle] = useState(false)
+
+    return (
+        <>
+            <table className={`table label-value-table ${styles['summary-table']}`}>
+                <thead
+                    title='Click to expand or collapse'
+                    onClick={() => {
+                        setToggle(!toggle)
+                    }}
+                >
+                    <tr>
+                        <th colSpan={2}>{t('Vulnerability Summary')}</th>
+                    </tr>
+                </thead>
+                <tbody hidden={toggle}>
+                    <tr>
+                        <td>{t('Title')}:</td>
+                        <td>{summaryData.title && summaryData.title}</td>
+                    </tr>
+                    <tr>
+                        <td>{t('Description')}:</td>
+                        <td>{summaryData.description && summaryData.description}</td>
+                    </tr>
+                    <tr>
+                        <td>{t('External Id')}:</td>
+                        <td>{summaryData.externalId && summaryData.externalId}</td>
+                    </tr>
+                    <tr>
+                        <td>{t('Publish Date')}:</td>
+                        <td>{summaryData.publishDate && summaryData.publishDate}</td>
+                    </tr>
+                    <tr>
+                        <td>{t('Last update')}:</td>
+                        <td>{summaryData.lastUpdateDate && summaryData.lastUpdateDate}</td>
```

Comment:
This should be **lastExternalUpdate** instead of **lastUpdateDate**

Reply:
Done